Packs/LogRhythmRest/Integrations/LogRhythmRest/LogRhythmRest.yml

```
@@ -609,60 +2469,1033 @@ script:
       description: The alarm web console IDs.
       type: String
     - contextPath: Logrhythm.Alarm.Summary.PIFType
-      description: Alarm Primary Inspection Field (the original name for "Summary Field").
+      description: Alarm Primary Inspection Field (the original name for "Summary
+        Field").
       type: String
     - contextPath: Logrhythm.Alarm.Summary.DrillDownSummaryLogs
       description: Drill down summary logs.
       type: String
-  - arguments:
-    - default: false
-      description: The alarm ID.
-      isArray: false
-      name: alarm-id
+    description: Returns data for an alarm.
+  - name: lr-get-alarm-events
+    arguments:
+    - name: alarm-id
       required: true
-      secret: false
-    - default: false
-      defaultValue: '10'
+      description: The alarm ID.
+    - name: count
       description: Number of events to return. Default is 10.
-      isArray: false
-      name: count
-      required: false
-      secret: false
-    - default: false
-      description: A CSV list of fields (outputs) to return to the context. If empty, all fields are returned.
-      isArray: false
-      name: fields
+      defaultValue: "10"
+    - name: fields
+      auto: PREDEFINED
       predefined:
-      - ''
-      required: false
-      secret: false
-    - auto: PREDEFINED
-      default: false
-      defaultValue: 'False'
-      description: Returns the log message from the event.
-      isArray: false
-      name: get-log-message
+      - ""
+      description: A CSV list of fields (outputs) to return to the context. If empty,
+        all fields are returned.
+    - name: get-log-message
+      auto: PREDEFINED
       predefined:
-      - 'True'
-      - 'False'
-      required: false
-      secret: false
-    deprecated: false
-    description: Returns a list of events, by alarm ID.
-    execution: false
-    name: lr-get-alarm-events
+      - "True"
+      - "False"
+      description: Returns the log message from the event.
+      defaultValue: "False"
     outputs:
     - contextPath: Logrhythm.Alarm.Event
       description: Alarm event information.
       type: String
     - contextPath: Logrhythm.Alarm.ID
       description: The alarm ID.
       type: String
-  isfetch: false
+    description: Returns a list of events, by alarm ID.
+  - name: lr-search-data
+    arguments:
+    - name: number-of-date
+      description: Number of dates to search
+    - name: source-type
+      auto: PREDEFINED
+      predefined:
+      - API - AWS CloudTrail
```

```suggestion
      - API_-_AWS_CloudTrail
```
The values should not have a space inside please update all of them